pkg/antctl/fallback/version/fallback.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2021 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package version
+
+import (
+	"io"
+	"strings"
+)
+
+func RequestErrorFallback() (io.Reader, error) {
+	return strings.NewReader("{}"), nil
```

Comment:
Is that the Transform func will still be called for the fallback response?

Reply:
correct, I considered several approaches but that's the one I went for: generate a "fake" server response and let it go through the transform functions as usual. An alternative was to modify the transforms to handle `nil` responses, but that seemed more error-prone, more invasive and less flexible.